deployment/services/db-migrations.ts

```diff
@@ -60,6 +60,7 @@ export function deployDbMigrations({
         CLICKHOUSE_USERNAME: clickhouse.config.username,
         CLICKHOUSE_PASSWORD: clickhouse.config.password,
         CLICKHOUSE_PROTOCOL: clickhouse.config.protocol,
+        CLICKHOUSE_MIGRATOR_GRAPHQL_HIVE_CLOUD: '1',
```

Comment:
this is something that self-hosted users will need to have as well? do we want to have some strategy to support self-hosted as well?

Reply:
What do you mean? The `CLICKHOUSE_MIGRATOR_GRAPHQL_HIVE_CLOUD` is to tell the migration script that it runs within GraphQL Hive. Self-hosters have it `false` by default.